class FAException(Exception):
    pass


class CoverageError(FAException):
    pass
